src/Api/Controllers/GroupsController.cs

```diff
@@ -56,7 +66,14 @@ public async Task<GroupResponseModel> Get(string orgId, string id)
     public async Task<GroupDetailsResponseModel> GetDetails(string orgId, string id)
     {
         var groupDetails = await _groupRepository.GetByIdWithCollectionsAsync(new Guid(id));
-        if (groupDetails?.Item1 == null || !await _currentContext.ManageGroups(groupDetails.Item1.OrganizationId))
```

Comment:
nit: I would love to get rid of the .Item1 by switching to a value tuple with named elements. I know it's pre-existing but from a reviewing standpoint I just have to infer what I think that is without looking it up.

Reply:
Ha, usually it's me telling other people to do this 😁 done just so it's done.